resources/examples/manager/SampleAssetManager/tests/test_apiCompliance.py

```diff
@@ -0,0 +1,67 @@
+#
+#   Copyright 2013-2022 The Foundry Visionmongers Ltd
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+"""
+Test cases for the SampleAssetManager that make use of the OpenAssetIO
+manager test harness.
+"""
+
+# pylint: disable=no-self-use
+# pylint: disable=invalid-name,redefined-outer-name
+# pylint: disable=missing-class-docstring,missing-function-docstring
+
+import os
+import pytest
+
+from openassetio.test.manager import harness, apiComplianceSuite
+
+#
+# Tests
+#
+
+class Test_SampleAssetManager:
+
+    def test_passes_apiComplianceSuite(self, api_compliance_fixtures):
+        assert harness.executeSuite(apiComplianceSuite, api_compliance_fixtures)
+
+#
+# Fixtures
+#
+
+@pytest.fixture(autouse=True)
+def sam_plugin_env(sam_base_dir, monkeypatch):
+    """
+    Provides a modified environment with the SampleAssetManager
+    plugin on the OpenAssetIO search path.
+    """
+    plugin_dir = os.path.join(sam_base_dir, "python")
+    monkeypatch.setenv("OPENASSETIO_PLUGIN_PATH", plugin_dir)
+
+@pytest.fixture
```

Comment:
[consider] Here and below - PEP8 - 2 lines in-between functions.

Reply:
Good spot, thanks, updated [here](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/219/commits/09108bfbde0752c871b43c782da29ca4e868de52).